app/api/server/index.js

```diff
@@ -18,6 +18,7 @@ import './v1/e2e';
 import './v1/emoji-custom';
 import './v1/groups';
 import './v1/im';
+import './v1/inquiries';
```

Comment:
Let's move your code to the right place, ok?
Since it's a livechat api implementation, please move this code to:
https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/api/rest.js